### DISCUSSION TITLE
Help me pls!!! i dont know how to design the data structure or template.

### DISCUSSION THREAD

**Discussion by h4p3** [jxlsteam/jxls#270] [Q&A] [answered]
i want this:
![image](https://github.com/jxlsteam/jxls/assets/20726602/2a9dd5e1-1217-41f3-bc17-98c94ba937e0)
it's my data structure:
![image](https://github.com/jxlsteam/jxls/assets/20726602/48ef9f31-a301-47c4-be54-e75cf1c0e549)
and my design template:
![image](https://github.com/jxlsteam/jxls/assets/20726602/ae6690db-372d-46d9-8834-4f4e487862fe)
and it's result,not i wanted result:
![image](https://github.com/jxlsteam/jxls/assets/20726602/a23a31f7-f9db-4c8b-aa63-5a5e7e063f86)
what should i do? pls help me. give me some advice.

**Comment by SoltauFintel** [comment, marked as answer]
a) "alisaName" looks like a typo.

b) You need the direction=right notes in each row. Also in row 3 and row 4.

c) Instead of $[SUM(..)] you could also try normal Excel sums.

**Reply by h4p3** [reply]
thanks reply, the b) i need the dynamic cols. i need traverse columns while traversing rows.

**Reply by SoltauFintel** [reply]
Is this a question (if yes: see my solution above) or can the discussion be closed?
